spring-cloud-sleuth-autoconfigure/src/main/java/org/springframework/cloud/sleuth/autoconfig/instrument/jdbc/TraceJdbcProperties.java

```diff
@@ -190,7 +190,7 @@ public void setJsonFormat(boolean jsonFormat) {
 			/**
 			 * Enable logging all queries to the log.
 			 */
-			private boolean enableLogging = false;
+			private boolean enableLogging = true;
```

Comment:
We had a chat with @gavlyukovskiy that having this turned on by default can lead to gigantic amount of logs. I'd leave both as opt in.

Reply:
So in that case we should set this [DataSourceProxyProperties.java#L116](https://github.com/spring-cloud/spring-cloud-sleuth/blob/3.1.x/spring-cloud-sleuth-instrumentation/src/main/java/org/springframework/cloud/sleuth/instrument/jdbc/DataSourceProxyProperties.java#L116) to `false` too, because at the end we are using this property class to build `ProxyDataSource`.

Also, if we set it to `false`, we have to modify jUnit as well, because then by default `SLF4JQueryLoggingListener.class` won't be registered.

So by default we have to enable `SLF4JSlowQueryListener` and `SLF4JQueryLoggingListener` should be opt-in feature.

Is this understanding correct ?

Reply:
I think that we want to set all the default additional loggings to `false` and make both opt in.

Reply:
Got it, will make all the `enableLogging=false` means no `QueryLogListener` or `SlowQueryLogListener` will be registered by default.